### PR TITLE
10.0 crm claim

### PR DESCRIPTION
**Pull request by techspawn**
crm_claim model for rma

### CODE REVIEW

**Comment by pedrobaeza** [comment]
This still includes all the modules.

**Comment by techspawn** [comment]
@pedrobaeza Please check and suggest the changes that need to be implemented

**Comment by pedrobaeza** [comment]
Please squash all the current commits to only one for starting with.

File: crm_claim_rma/models/claim_line.py [hunk 1]
```diff
@@ -115,20 +116,18 @@ class ClaimLine(models.Model):
...
         "delivery or default address and so on for supplier and brand "
         "manufacturer. Does not necessarily mean that the warranty "
         "to be applied is the one of the return partner (ie: can be "
         "returned to the company and be under the brand warranty")
-    warranty_return_partner = \
-        fields.Many2one('res.partner', string='Warranty Address',
+    warranty_return_partner = fields.Many2one('res.partner', string='Warranty Address',
                         help="Where the customer has to "
                         "send back the product(s)")
     claim_id = fields.Many2one('crm.claim', string='Related claim',
```
Comment (yvaucher) [inline crm_claim_rma/models/claim_line.py]: model `crm.claim` does not exist anymore in Odoo 10.0 community edition

**Review by yvaucher — CHANGES_REQUESTED** [review]

**Comment by bodedra** [comment]
I would like to contribute here. Would you please suggest me How can I do that?

**Comment by max3903** [comment]
@techspawn Please review https://github.com/TechspawnSolutions/rma/pull/1

**Comment by max3903** [comment]
@bodedra We are still missing the migration of a dependency: crm_claim_rma_code

**Comment by max3903** [comment]
Waiting on #126

**Comment by max3903** [comment]
Superseeded in #128

**Comment by gurneyalex** [comment]
@max3903 why did you reopen this PR ? is it still superseeded by #128 ?

**Comment by gurneyalex** [comment]
Actually superseeded by https://github.com/OCA/crm/pull/170

**Pull request closed without merging**